Turn 'deployment considerations' into 'deployment path' (#3456)
theforeman/foreman-documentation: guides/common/modules/con_adding-a-red-hat-subscription-to-your-project-server.adoc

[id="adding-a-red-hat-subscription-to-your-project-server-{context}"]

> = Adding a Red{nbsp}Hat subscription to your {ProjectServer}

Reviewer: I am unsure about "adding" and "{ProjectServer}". The concept is about importing a RH manifest to {Project}, not adding a valid RHEL subscription to your Foreman Server.

With ACS, one could IMHO argue that even Smart Proxy Servers access RH CDNs to get packages.

Author: Addressed (including renaming the file etc.)

A Red{nbsp}Hat subscription manifest is a set of encrypted files that contains your subscription information.

> {ProjectServer} uses this information to access the CDN and find what repositories are available for the associated subscription.

Reviewer: ```suggestion
{ProjectServer} uses this information to access the Red{nbsp}Hat CDN and find what repositories are available for the associated subscription.
```

[WARNING]
====
Deleting a subscription manifest removes all the subscriptions attached to running hosts and activation keys.
====

.Additional resources

> * For instructions about creating and importing a Red{nbsp}Hat subscription manifest see {ContentManagementDocURL}Managing_Red_Hat_Subscriptions_content-management[Managing Red Hat Subscriptions] in _{ContentManagementDocTitle}_.

Reviewer: ```suggestion
* For instructions about creating and importing a Red{nbsp}Hat subscription manifest, see {ContentManagementDocURL}Managing_Red_Hat_Subscriptions_content-management[Managing Red Hat subscriptions] in _{ContentManagementDocTitle}_.
```

> * For details about using subscription manifests with {Project}, see link:https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/creating_and_managing_manifests_for_a_connected_satellite_server/index[_Creating and managing manifests for a connected Satellite Server_].

Reviewer: ```suggestion
* For details about using subscription manifests with {Project}, see {RHDocsBaseURL}/subscription_central/1-latest/html/creating_and_managing_manifests_for_a_connected_satellite_server/index[_Creating and managing manifests for a connected Satellite Server_].
```

source: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L28
